Gemfile.lock

```diff
@@ -1,14 +1,94 @@
 PATH
   remote: .
   specs:
-    bump (0.5.3)
+    bump (0.5.12)
 
 GEM
   remote: https://rubygems.org/
   specs:
+    addressable (2.4.0)
+    berkshelf (4.3.5)
```

Comment:
I'll see what I can do to mock out berkshelf in the tests to remove the dependencies. However these would only be development/test dependencies. Any project using bump would not be required to install this.